pkg/controller/common/constants.go

```diff
@@ -30,4 +30,22 @@ const (
 
 	// ClusterFeatureInstanceName is a singleton name for featureGate configuration
 	ClusterFeatureInstanceName = "cluster"
+
+	// ExperimentalLayeringPoolLabel is the label that enables the "layered" workflow path for a pool
+	ExperimentalLayeringPoolLabel = "machineconfiguration.openshift.io/layered"
+	// ExperimentalNewestLayeredImageAnnotationKey is the annotation that signifies the newest image that has been rendered to a machine
+	// config pool's imagestream
+	ExperimentalNewestLayeredImageAnnotationKey = "machineconfiguration.openshift.io/newestImage"
+	// ExperimentalNewestLayeredImageEquivalentConfigAnnotationKey is the annotation that signifies which rendered config
+	// the currently rendered image is equivalent to
+	ExperimentalNewestLayeredImageEquivalentConfigAnnotationKey = "machineconfiguration.openshift.io/newestImageEquivalentConfig"
+
+	// ImageStreamSuffixRenderedConfig is the suffix for a pool imagestream where the mco also writes its rendered machineconfig
+	ImageStreamSuffixRenderedConfig = "-rendered-config"
+	// ImageStreamSuffixMcoContent  is the suffix for a pool imagestream where the mco has applied its rendered-config to the base image
+	ImageStreamSuffixMcoContent = "-mco-content"
+	// ImageStreamSuffixMcoContentCustom is the suffix for a pool imagestream where content has been customized by a user
+	ImageStreamSuffixMcoContentCustom = "-mco-content-custom"
+	// ImageStreamSuffixMcoContentExternal is the suffix for a pool imagestream  where content comes from outside the cluster
+	ImageStreamSuffixMcoContentExternal = "-mco-content-external"
```

Comment:
:thinking: wondering if MCO would be better than Mco

Reply:
I capitalized them all in case we keep them.